typescript/appconfig-hosted-configuration-lambda-extension/index.ts

```diff
@@ -0,0 +1,89 @@
+import * as cdk from '@aws-cdk/core';
+import { CfnApplication, CfnEnvironment, CfnConfigurationProfile, CfnHostedConfigurationVersion, CfnDeploymentStrategy, CfnDeployment } from '@aws-cdk/aws-appconfig';
+import { Function, AssetCode, Runtime, LayerVersion } from '@aws-cdk/aws-lambda';
+import { Effect, PolicyStatement } from '@aws-cdk/aws-iam';
+
+export class AppConfigHostedConfigurationStack extends cdk.Stack {
+	constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
+		super(scope, id, props);
+
+		const application = new CfnApplication(this,'AppConfigApplication', {
+			name: 'AppConfigSampleApplication',
+			description: 'Sample AppConfig Application using CDK'
+		});
+
+		const environment = new CfnEnvironment(this, 'LambdaDevelopmentEnvironment', {
```

Comment:
It would be useful here to include both a dev and prod environment with appropriate configurations for each. This would show the real power/reason to use AppConfig over a standard deployment strategy.